README.md

```diff
@@ -20,10 +20,25 @@ To provision services, the brokerpak currently requires GCP credentials. The bro
 
 A Makefile supports the full local development lifecycle for the brokerpak.
 
-Make targets will run with the *cfplatformeng/csb* docker image. Alternatively, a custom image can be specified by setting the `CSB` environment variable
+The make targets can be run either with Docker or installing the required libraries in the local OS.
 
 Available make targets can be listed by running `make`.
 
+### Running with docker
+
+1. Install [Docker](https://docs.docker.com/get-docker/)
+2. If you don't have GoLang installed, the makefile will automatically use Docker. If you do have go installed but still want to use docker, then set the `USE_GO_CONTAINERS` to `true`.
+
+Make targets will run with the *cfplatformeng/csb* docker image. Alternatively, a custom image can be specified by setting the `CSB` environment variable.
+
+### Running with GoLang
```

Comment:
https://go.dev/doc/faq#:~:text=any%20old%20gopher.-,Is%20the%20language%20called%20Go%20or%20Golang%3F,site%20was%20originally%20golang.org.

There's no such thing as "GoLang"